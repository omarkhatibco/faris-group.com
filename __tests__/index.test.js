/* eslint-env jest */

import React from 'react';
import { render } from '@testing-library/react';

import Home from '../pages/index.tsx';

describe('With React Testing Library', () => {
	it('Shows "Hello world!"', () => {
		const { getByText } = render(<Home />);

		expect(getByText('Welcome to Next!')).not.toBeNull();
	});
});

describe('With React Testing Library Snapshot', () => {
	it('Should match Snapshot', () => {
		const { asFragment } = render(<Home />);

		expect(asFragment()).toMatchSnapshot();
	});
});
