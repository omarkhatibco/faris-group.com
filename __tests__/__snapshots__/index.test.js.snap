// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With React Testing Library Snapshot Should match Snapshot 1`] = `
<DocumentFragment>
  <main
    class="css-17calk8"
  >
    <div
      class="css-13wevqw"
    >
      <div
        class="css-1gy22ay"
      >
        Loading...
      </div>
    </div>
    Home
  </main>
</DocumentFragment>
`;
